letta/server/rest_api/utils.py

```diff
@@ -45,6 +51,19 @@ async def sse_async_generator(generator: AsyncGenerator, finish_message=True):
                 chunk = str(chunk)
             yield sse_formatter(chunk)
 
+        # If we have a usage task, wait for it and send its result
+        if usage_task is not None:
```

Comment:
If the `server.send_messages` call returned, then the value of `usage_task` here should resolve to `LettaUsageStatistics`